ui/raidboss/data/05-shb/dungeon/dohn_mheg.js

```diff
@@ -102,24 +82,7 @@
       regexJa: Regexes.startsUsing({ id: '2299', source: '美眼のインク＝ゾン' }),
       regexCn: Regexes.startsUsing({ id: '2299', source: '美眼 因克·佐恩' }),
       regexKo: Regexes.startsUsing({ id: '2299', source: '눈이 예쁜 잉크 돈' }),
-      alertText: function(data, matches) {
-        if (matches[1] == data.me) {
-          return {
-            en: 'Tank Buster on YOU',
-            de: 'Tankbuster auf DIR',
-            fr: 'Tankbuster sur VOUS',
-            ko: '나에게 탱버',
-          };
-        }
-        if (data.role == 'healer') {
-          return {
-            en: 'Buster on ' + data.ShortName(matches[1]),
-            de: 'Tankbuster auf ' + data.ShortName(matches[1]),
-            fr: 'Tankbuster sur ' + data.ShortName(matches[1]),
-            ko: data.ShortName(matches[1]) + '에게 탱버',
-          };
-        }
-      },
+      response: Responses.tankBuster(),
```

Comment:
tankbuster condition :skull_and_crossbones:

Reply:
i hope i did not overlook any now Q_Q